.flowconfig

```diff
@@ -1,4 +1,5 @@
 [ignore]
+.*/node_modules/@babel/core
```

Comment:
This prevents the following flow error: https://github.com/paypal/paypal-checkout-components/runs/1559689002?check_suite_focus=true#step:6:93